git/foundations_git/git_basics.md

```diff
@@ -124,6 +124,10 @@ Finally, let's upload your work to the GitHub repository you created at the star
     <a href="https://cdn.statically.io/gh/TheOdinProject/curriculum/b54d14c5dcee1c6fac61aee02fca7e9ef7ba1510/foundations/git_basics/project_practicing_git_basics/imgs/20.png"><img class="tutorial-img" src="https://cdn.statically.io/gh/TheOdinProject/curriculum/b54d14c5dcee1c6fac61aee02fca7e9ef7ba1510/foundations/git_basics/project_practicing_git_basics/imgs/20.png" title="Verify repo changes are on GitHub" /></a>
 </div>
 
+### Note/warning
+
+When trying to make simple changes to the files in your repo, such as attempting to fix a typo in your README.md you might be tempted to make this change directly via Github. However, it is best to avoid this as it will cause issues with your branches diverging and merging/resetting them is too complicated at this stage (it is covered in a future lesson), for now it is advised to make any changes via your local files then commit and push them using git commands in your terminal once ready.
```

Comment:
```suggestion
When trying to make simple changes to the files in your repo, such as attempting to fix a typo in your README.md you might be tempted to make this change directly via Github. However, it is best to avoid this as it will cause issues that require more advanced Git knowledge than we want to go over at this stage (it is covered in a future lesson), for now it is advised to make any changes via your local files then commit and push them using Git commands in your terminal once ready.
```